src/components/Link/index.tsx

```diff
@@ -20,7 +20,13 @@ export const Link: React.FunctionComponent<ILink> = ({ to, children, disabled, .
   } else if (typeof href === 'string') {
     href = href.trim(); // Make sure there aren't any trailing white spaces
 
-    if (/^http/.test(href)) {
+    if (href.includes('/p/')) {
+      return (
+        <a {...props} href={href} target="_blank">
+          {children}
+        </a>
+      );
+    } else if (/^http/.test(href)) {
```

Comment:
Let's combine these two conditionals

```suggestion
    if (href.includes('/p/') || /^http/.test(href)) {
```